### PR TITLE
Refactoring: Deduplication

### PR DESCRIPTION
**Pull request by marload**
I improved the quality of code by removing the duplication of code.

### CODE REVIEW

File: metric-collector/service-readiness/kubeflow-readiness.py [hunk 1]
```diff
@@ -29,12 +29,9 @@ def metric_update(args, google_open_id_connect_token):
       headers={
           'Authorization': 'Bearer {}'.format(google_open_id_connect_token)
       })
-  if resp.status_code == 200:
-    KUBEFLOW_AVAILABILITY.set(1)
-    return 1
-  else:
-    KUBEFLOW_AVAILABILITY.set(0)
-    return 0
+  v = 1 if resp.status_code == 200 else 0
```
Comment (jlewi) [inline metric-collector/service-readiness/kubeflow-readiness.py:32]: nit: please put if and else on separate lines I think its more readable then this comprehension

Comment (marload) [reply metric-collector/service-readiness/kubeflow-readiness.py]: Thank You! I applied this!

**Pull request merged**